pkg/model/awsmodel/api_loadbalancer.go

```diff
@@ -174,11 +175,15 @@ func (b *APILoadBalancerBuilder) Build(c *fi.ModelBuilderContext) error {
 	// Create security group for API ELB
 	var lbSG *awstasks.SecurityGroup
 	{
+		removeExtraRules := []string{"port=443"}
+		if lbSpec.SSLCertificate != "" {
+			removeExtraRules = append(removeExtraRules, "port=8443")
```

Comment:
We should probably do this even when `lbspec.SSLCertificate` is empty. Otherwise will disabling the option clear the extra rule for 8443?

Reply:
good call, I've updated it to always remove extra rules on 8443 too.